dongken/app.js

```diff
@@ -44,11 +44,38 @@ app.post('/users/signup', async (req, res) => {
             password
 		) VALUES (?, ?, ?, ?);
 		`,
-		[ name, email, profile_image, password]
+		[ name, email, profile_image, bcryptPassword]
```

Comment:
동근님, 탭, 띄어쓰기를 동시에 사용할 경우 indentation이 어긋나는 경우가 발생할 수 있습니다. git push 를 하고 난 후에 포맷팅이 잘 되어 있는지 확인해 보시는 걸 추천드립니다!
그리고 아래와 같이 수정해 주세요!
```suggestion
await appDateSource.query(
    `INSERT INTO users(
            name,
            email,
            profile_image,
            password
    ) VALUES (?, ?, ?, ?);`,
[ name, email, profile_image, bcryptPassword]
```